### PR TITLE
[#2615] Fix sample.timeout() being invoked for each request

### LINKED ISSUE #2615: Timeout metric gets increased with every request
When sending a request, which expects a response, using an AMQP client based on the Hono `RequestResponseClient` or `AbstractRequestResponseClient` classes, each request increases the `timeout` metric.

### PR DESCRIPTION
**Pull request by calohmn**
This fixes #2615.

### CODE REVIEW

**Review by sophokles73 — COMMENTED** [review]
Is there also a unit test validating the behavior?

**Review by sophokles73 — APPROVED** [review]
LGTM

Can you also please add en entry to the release notes? And pick the commit on the 1.7.x branch? Would be good if we could include this in 1.7.1.

File: site/homepage/content/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ description = "Information about changes in recent Hono releases. Includes new f
 ### Fixes & Enhancements
 
 * The CoAP adapter did not correctly track the time it took to forward a command message to a device. This has been fixed.
+* Sending requests using the Hono AMQP request-response client erroneously increased the `timeout` metric. This has
```
Comment (sophokles73) [inline site/homepage/content/release-notes.md:12]: it is actually the `hono.downstream.timeout` metric ...

**Review by sophokles73 — APPROVED** [review]
LGTM

**Comment by calohmn** [comment]
Committed in 1.7.x as well.

**Pull request merged**